dcos_launch/platforms/gcp.py

```diff
@@ -82,23 +85,44 @@
   targetSize: {size}
 """
 
-# template for a firewall in the network of a gce deployment
-FIREWALL_TEMPLATE = """
+# template for a firewall that controls external access
+EXTERNAL_FIREWALL_TEMPLATE = """
 type: compute.v1.firewall
-name: {name}
+name: {name}-external
 metadata:
   dependsOn:
   - {network}
 properties:
-  description: allow all ports
+  description: external
   network: global/networks/{network}
   sourceRanges:
   - 0.0.0.0/0
   allowed:
   - IPProtocol: tcp
-  - IPProtocol: udp
+    ports:
+    - 22
+    - 80
+    - 443
+    - 9000
```

Comment:
Is this for the web installer (I.E. the installer server)? If so, we no longer user it in dcos-launch and can safely remove